src/TagsInput/components/InputField.js

```diff
@@ -77,47 +98,50 @@ class InputField extends React.PureComponent<Props, State> {
       forwardRef,
       onFocus,
       onBlur,
-      autoFocus,
+      onKeyPress,
+      autofocus,
     } = this.props;
     const { componentWidth } = this.state;
-    const dynamicStyle = StyleSheet.create({
-      common: {
-        minWidth,
-        maxWidth,
-        fontSize,
-      },
+
+    const dynamicStyle = {
+      common: { fontSize },
       input: {
         width: componentWidth,
       },
-    });
+    };
+
+    const sizerValue = value || (placeholder || '');
 
     return (
       <View style={styles.container}>
-        <Text
-          onLayout={this.handleOnLayout}
-          style={[styles.hiddenSizer, dynamicStyle.common]}
-        >
-          {!value && placeholder ? placeholder : value}
-        </Text>
         <TextInput
+          autoFocus={autofocus}
+          onKeyPress={onKeyPress}
           onBlur={onBlur}
           onFocus={onFocus}
           ref={forwardRef}
           editable={!disabled}
-          autoFocus={autoFocus}
           onChangeText={onChangeText}
           value={value}
           placeholder={placeholder}
-          style={[styles.input, dynamicStyle.common, dynamicStyle.input]}
+          style={[styles.input, dynamicStyle.input, dynamicStyle.common]}
         />
+        <Text
```

Comment:
So we're displaying something that's opacity 0? I am not sure this is going to be great for accessibility.

Reply:
Me either, but there is nothing like `visibility: 'hidden',` for react-native
but I will add there `accessibilityRole="none"` this should probably fix this issue.